cmd/deployments.go

```diff
@@ -41,13 +42,20 @@ var (
 		Name:     "deployments",
 		Category: "Deployment",
 		Usage:    "List app deployments",
-		Flags:    []cli.Flag{&appFlag},
+		Flags: []cli.Flag{
+			&appFlag,
+			&cli.IntFlag{Name: "page", Usage: "Page to display", Value: 1},
+			&cli.IntFlag{Name: "per-page", Usage: "Number of deployements to display", Value: 30},
```

Comment:
Out of curiosity why do use 30 per page by default as the default value per page value is 20 on the API?

Reply:
Because the only other feature using pagination was configured with 30 by default